src/common/stmt-syntax.k

```diff
@@ -25,12 +26,13 @@ syntax StmtWithoutTrailing ::= Block
 |SynchronizedStmt
 
 syntax BlockStmt ::= Stmt|LocalVarDecStmt|ClassDec
-syntax LocalVarDecStmt ::= Type Id ";" [klabel('LocalVarDec)]
-syntax ClassBody ::= "{" KListWrap "}" [klabel('ClassBody)]
-syntax ClassDecHead ::= KListWrap "class" Id K K K [klabel('ClassDecHead)]
+syntax LocalVarDecStmt ::= LocalVarDec ";" [klabel('LocalVarDecStm)]
+syntax LocalVarDec ::= AnnoVarModList Type VarDecList [klabel('LocalVarDec)]
+syntax ClassBody ::= "{" ClassBodyDecList "}" [klabel('ClassBody)]
+syntax ClassDecHead ::= AnnoClassModList "class" Id K K K [klabel('ClassDecHead)]
 syntax ClassDec ::= ClassDecHead ClassBody [klabel('ClassDec)]
 
-syntax Block ::= "{" BlockStmt "}" [klabel('Block)]
+syntax Block ::= "{" K "}" [klabel('Block)]
```

Comment:
Most rules are in KLabel format. But {} is special, some rules are already written in concrete syntax. like this:
try TryS:K (K:CatchClause Ks:CatchClauses) finally FinallyS:K
    => try {try TryS (K Ks)} .CatchClauses finally FinallyS

I am debugging why the computation stucks at try catch when I change from K to BlockStmList.